.github/workflows/tests.yml

```diff
@@ -0,0 +1,26 @@
+name: CI/CD Tests
+   
+on: [push]
+
+jobs:
+  build:
```

Comment:
I believe _test_ is a better term here.
We're not undertaking a build step here.

Reply:
Ah, that job name slipped through while repurposing an example workflow. Renamed to _test_.